### DISCUSSION TITLE
How to handle print styles after upgrading to V5

### DISCUSSION THREAD

**Discussion by MightyMoud** [twbs/bootstrap#34553] [Q&A] [answered]
Hi guys,
We have an app that uses bootstrap. We have one page that gets captured by puppeteer into a pdf. This was working very nicely on v4.6. After upgrading to V5, styles are all messed up on that pdf now. 

I read somewhere here that V5 doesn't include print scss file anymore. Then how can we handle applying the sane style to the print version? We don't need anything special, just the way you see it in the browser. 
Any help?

**Comment by coliff** [comment, marked as answer]
I have just the thing you need!
https://github.com/coliff/bootstrap-print-css

If you simply want to copy and paste the CSS to your existing CSS you can do so with this:
https://github.com/coliff/bootstrap-print-css/blob/main/css/bootstrap-print.css
